packages/antlr4.rb

```diff
@@ -20,15 +20,14 @@ def self.install
     system 'curl -#LO https://www.antlr.org/download/antlr-4.7.1-complete.jar'
```

Comment:
New version?  See https://www.antlr.org/download/antlr-4.9.2-complete.jar.

Reply:
again, could we upgrade after?

Reply:
My thinking is if you are going to change it, might as well upgrade.

Reply:
This should be an easy upgrade, right?  No binaries necessary.